webapp/src/client/app/modules/web-client/vnc/web-client-vnc.component.ts

```diff
@@ -400,4 +398,32 @@ export class WebClientVncComponent extends WebClientBaseComponent implements  On
         return 'Connection error: Please verify your connection settings.';
     }
   }
+
+  private getErrorKind(errorData: UserIronRdpError | string): UserIronRdpErrorKind {
+    if (typeof errorData === 'string') {
+      return this.parseErrorString(errorData);
+    }
```

Comment:
question: Does it make sense to parse strings not returned by the WASM module? (I’m not sure of the provenance of the "string errors", but that’s not returned by the WASM module at least.)

Reply:
It seems the web component may return strings then, but you don’t have to parse these. It’s likely for web component-specific errors.

Reply:
@CBenoit Ok. I will not parse them. Thank you!